test/test_ops.py

```diff
@@ -287,6 +287,14 @@ def _test_consistency_helper(samples, variants):
                     # Note: copies the to-be-modified input when testing the inplace variant
                     tensor.grad = None
                     cloned = clone_input_helper(sample.input) if variant in inplace_ops else sample.input
+
+                    if variant in inplace_ops and sample.broadcasts_input:
+                        with self.assertRaises(RuntimeError):
```

Comment:
Received two different error messages, so haven't used `assertRaisesRegex`.

Message 1:
```
expand(CUDABoolType{[5, 5]}, size=[5]): the number of sizes provided (1) must be greater or equal to the number of dimensions in the tensor (2)
```

Message 2:
```
output with shape [5, 5] doesn't match the broadcast shape [5, 5, 5]
```

Reply:
Not using assertRaisesRegex sounds good